source/process/release-process.md

```diff
@@ -164,9 +162,6 @@ Day when Leads and PMs decide which major features are included in the release,
 3. QA:
     - Update Release Discussion header with links to RC instances and testing spreadsheet ([template](https://pre-release.mattermost.com/core/pl/db3sur4r53d9tyih1i4wrmi9wy))
     - Post release testing instructions to Release Discussion channel ([template](https://pre-release.mattermost.com/core/pl/uprogtcqzpbk7nkmdkfnhqkcac))
-    - Update Selenium and manual smoke tests as needed, to prepare to run on each RC after it is cut
```

Comment:
Looks like on `master` we also have `Post "Bug Hunter Coin" message to Reception channel ([see example](https://pre-release.mattermost.com/core/pl/3o15eoq89fdq5m1ac5dyp4nc3e))`.

Do we want to keep that item?

Reply:
Oh yes @jasonblais. I missed that one (was working on an older branch of mine). What's the best way to do that?

Reply:
Thanks @jasonblais! Added it back in. Thanks for catching that.